### ISSUE TITLE
Python 3 issue with saving Tasks app state

### ISSUE THREAD

**Issue by corranwebster** [enthought/envisage#74]
Reported by Eric McDonald:

```
File "C:\Anaconda3\lib\site-packages\envisage\ui\tasks\tasks_application.py", line 315, in _load_state 
restored_state = pickle.load(f) 
TypeError: a bytes-like object is required, not 'str'
```

and similar errors.  Reported fix is that files need to be opened in `rb` mode (which I think is bug in Python 2.7 as well - things probably aren't working as expected under Windows in some situations!)

**Comment by mdickinson** [comment]
This is out of date; not sure which PR it was fixed in. Closing.

**Comment by mdickinson** [comment]
It was fixed in #90.
